src/withZapp/index.js

```diff
@@ -1,26 +1,34 @@
 import React from 'react';
 import { Platform } from 'react-native';
 
-const propsToParse = ['localization', 'settings', 'styles', 'extra_props'];
+const propParsers = {
+  localization: { android: JSON.parse },
+  settings: { android: JSON.parse },
```

Comment:
@dickie81 I missed this somehow - would you wrap this in try catch or something ? 
this way, when we (finally) implement writableMaps on android, we won't have to parse props anymore

with ramda you can do `tryCatch(JSON.parse, flip(identity))`